### PR TITLE
Get-CodeOwners - Hotfixes

### PR DESCRIPTION
**Pull request by scbedd**

### CODE REVIEW

File: eng/common/scripts/get-codeowners.ps1 [hunk 1]
```diff
@@ -26,15 +25,19 @@ foreach ($contentLine in $codeOwnersContent) {
   }
 }
 
-$results = $ownedFolders[$TargetDirectory.ToLower()]
+$matchedKey = $ownedFolders.keys `
+  | ? { $target.StartsWith($_) } `
+  | Sort-Object -Property Length -Descending `
```
Comment (weshaggard) [inline eng/common/scripts/get-codeowners.ps1:30]: Don't you want the shortest match? In which case you wouldn't want the descending option.

Comment (scbedd) [reply eng/common/scripts/get-codeowners.ps1:30]: Negative sir! I want the longest match! More exact the better!

Comment (scbedd) [reply eng/common/scripts/get-codeowners.ps1]: We talked offline. Updates complete.

**Pull request merged**